tests/python/unit/test_binary_model.py

```diff
@@ -0,0 +1,42 @@
+from python-prediction-algo.glm-binary-classification import CustomPredictionAlgorithm
```

Comment:
had to rewrite the unit tests as the structure of the model changed, also added some clean up to the json files c029587f4d2e07f8607422625fefb4dacbd6f56b

Reply:
The unit tests are still unable to be run with a make unit-tests command.
In the import of the tests, you should delete python-lib references.
Instead of `from python-lib.generalized_linear_models.dku_glm import RegressionGLM`, you should have `from generalized_linear_models.dku_glm import RegressionGLM`